### PR TITLE
Creation components

### PR DESCRIPTION
**Pull request by ines-missoum**

### CODE REVIEW

File: src/components/connexionContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+interface IProps {
+  text: string,
+  image: string,
+}
+
+interface IState {
+  text: string,
+  image: string,
+  form: any
```
Comment (ines-missoum) [inline src/components/connexionContainer.tsx]: not sure about the any that should be a react component but this is not working. ideas ?

Comment (MartinCayuelas) [inline src/components/connexionContainer.tsx]: form: React.Component ? maybe?

File: src/components/connexionNav.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import logo from '../img/polytechLogo.svg';
+
+interface IState {
+  isSignInActive: boolean,
+  handleClickFct: any
```
Comment (MartinCayuelas) [inline src/components/connexionNav.tsx]: Maybe any should be change by void here

File: src/components/connexionNav.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+interface IProps {
+  isSignInActive: boolean,
+  handleClickFct: any
```
Comment (MartinCayuelas) [inline src/components/connexionNav.tsx]: and here, same as above

File: src/components/actorContainer/actorNavContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            </div>
+            <div className="row">
+              <h4 style={{ color: 'white', paddingTop: '3%' }}>{this.props.userName}</h4>
+            </div>
+          </div>
+
+          {/*nav*/}
+          <div className="row container justify-content-md-center" style={{ height: '65%'}}>
+            <ul className="nav flex-column ">
+              {this.props.routes.map(bloc => <ActorNavBloc blocList={bloc} key={'keyBlock'}/>)}
```
Comment (MartinCayuelas) [inline src/components/actorContainer/actorNavContainer.tsx]: a key should Be unique :)

File: src/components/actorContainer/actorNavBloc.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  blocList: Array<[string, string]>
+}
+
+class ActorNavBloc extends React.Component<IProps> {
+
+  render() {
+    return (
+      <ul className="decor-line justify-content-md-start" style={{ listStyleType: 'none', paddingLeft: '0%' }}>
+        {this.props.blocList.map(e =>
+          <li className="nav-item" key={'key'} >
```
Comment (MartinCayuelas) [inline src/components/actorContainer/actorNavBloc.tsx]: here also for the key

**Pull request merged**